helm/values.yaml

```
@@ -65,6 +65,10 @@ microservice-chart:
   envSecret:
     APPLICATIONINSIGHTS_CONNECTION_STRING: appinsights-connection-string
 
+    MYPAY_DB_HOST: db-host
+    MYPAY_DB_USER: db-mypay-login-username
+    MYPAY_DB_PASSWORD: db-mypay-login-password
```

it was not a requirement to give the possibility to have multiple DataSource, but OK.

let's use ENV names not related to MYPAY, let's talk of PiattaformaUnitaria instead
```suggestion
    PU_DB_HOST: db-host
    PU_DB_USER: db-mypay-login-username
    PU_DB_PASSWORD: db-mypay-login-password
```